README.md

```
@@ -1,61 +1,135 @@
-# Build the frontend (static SPA served by backend)
-cd frontend && bun run build
+Run the test suites with `cd backend && bun test` and `cd frontend && bun test`. After schema changes, generate migrations with `bun drizzle-kit generate` in `backend/` — never write them by hand.
 
-# Run all tests
-cd backend && bun test
-cd frontend && bun test
+## How it works
 
-# Type-check the backend
-cd backend && bun run typecheck
 ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)